src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String DELIMITER_COMMA = ",";
+    private static final String DELIMITER_SPACE = " ";
+    private static final String DELIMITER_EMPTY = "";
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = readFromFile(fromFileName);
+        writeToFile(toFileName, report);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        File fileToRead = new File(fromFileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileToRead))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                stringBuilder.append(" ").append(line);
```

Comment:
can we just do the processing here?
split line by comma, and calculate `buySum` and `supplySum`?